wgpu-hal/src/vulkan/instance.rs

```diff
@@ -557,6 +560,20 @@ impl crate::Instance<super::Api> for super::Instance {
             layers
         };
 
+        #[cfg(target_os = "android")]
+        let android_sdk_version = {
+            // See: https://developer.android.com/reference/android/os/Build.VERSION_CODES
+            let mut prop = android_properties::getprop("ro.build.version.sdk");
```

Comment:
Can we avoid panics in this code? For example, if something goes wrong, just log error and return 0.

Reply:
can just fallback to a version to zero and hope for the best yeah, in this case it'd also have the intended effect if disabling the use of timeouts.